src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -274,6 +274,40 @@
       max-width: 560px;
     }
 
+    & .playButtons {
```

Comment:
this CSS is partially duplicated on purpose for both old and new design, eventually we'll remove the `old` styles and it's easier to do that with duplications